pip/torch.file

```diff
@@ -0,0 +1,72 @@
+## INCLUDE cuda-flags
+
+%define cuda_arch_float $(echo %{cuda_arch} | tr ' ' '\\n' | sed -E 's|([0-9])$|.\\1|' | tr '\\n' ' ')
+%define tag v%{realversion}
+%define branch release/1.13
+
+%define source0 git+https://github.com/pytorch/pytorch.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&submodules=1&output=/%{n}-%{realversion}.tgz
+#Patch0: py3-torch-cpp-externsion-ppc64 - TODO: is it needed? 1st part doesn't apply
+
+BuildRequires: cmake gmake
```

Comment:
looks like it uses ninja https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d4402c/31369/externals/py3-torch/1.13.1-dd73c8cc29f8c7f523192f1a36e115dc/log
```
cmake -GNinja -DBLAS=OpenBLAS -DBUILD_CAUSE=UPSTREAMTRIGGER -DBUILD_CAUSE_UPSTREAMTRIGGER=true 
```